wafer/schedule/admin.py

```diff
@@ -169,6 +169,26 @@ def check_schedule():
     return True
 
 
+@cache_result('wafer_schedule_validate_schedule', 30)
+def validate_schedule():
+    """Helper routine to easily test if the schedule is valid"""
+    all_items = prefetch_schedule_items()
+    errors = []
+    if find_clashes(all_items):
+        errors.append('Clashes found in schedule.')
+    if find_duplicate_schedule_items(all_items):
+        errors.append('Duplicate schedule items found in schedule.')
+    if validate_items(all_items):
+        errors.append('Invalid schedule items found in schedule.')
+    if find_overlapping_slots():
+        errors.append('Overlapping slots found in schedule.')
+    if find_non_contiguous(all_items):
+        errors.append('Non contiguous slots found in schedule.')
+    if find_invalid_venues(all_items):
+        errors.append('Invalid venues found in schedule.')
+    return errors
+
+
```

Comment:
This was why I changed the caching period for this validation to a lot lower, but I guess we probably don't wanna cache it at all.

I considered refactoring `check_schedule` to call `validate_schedule`, but we want `check_schedule` to exit early, while `validate_schedule`, but we could have that as an `arg`/`kwarg` to `validate_schedule`, because I think multiple validation errors are useful in the editor, much like Django form validation works.